src/DonorProfiles/Shortcode.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Give\DonorProfiles;
+
+use Give\DonorProfiles\Model as DonorProfile;
+
+class Shortcode {
+
+	protected $donorProfile;
+
+	public function __construct() {
```

Comment:
@henryholtgeerts Do you think instead of loading class with `give` function we can pass a parameter to function with `DonorProfile` class type which autoloads class and then assign it to `donorProfile ` class properties